vc-worker.py

```diff
@@ -110,7 +183,7 @@ def process_deploy_action(conn, action, vc):
         machine = data.Machine.get_one_for_update({'_id': request.machine}, conn=conn)
         machine.state = MachineState.FAILED
         machine.save(conn=conn)
-        logger.debug('updating action to be finished...')
+        logger.debug('updating action to be finished [Exception]...')
```

Comment:
I'd vote for logging the exception as `repr(e)`, so that we have the information in log.